data/remnant missions.txt

```diff
@@ -1252,3 +1252,237 @@ mission "Remnant: Expanded Horizons Quarg 3"
 			`	"Our historical records just have a few pictures and videos of Quarg ships, along with a basic scan that doesn't provide more than the general outline of the ship. These scans," he says while tapping the databanks, "actually picked up detailed structural layouts and even identified a number of key components. It isn't nearly enough information to be able to duplicate them, but it is enough that we can start trying to figure out how they work.`
 			`	Upon arriving at <planet> they quickly thank you, hand you <payment>, and run back to their offices to finish prepping their material to publish and distribute.`
 
+
+
+mission "Remnant: Broken Jump Drive 1"
+	name "Broken Jump Drive Delivery"
+	description "You have been asked to deliver a broken jump drive to the researchers on <planet>."
+	minor
+	landing
+	to offer
+		has "remnant blood test pure"
+	source
+		government "Remnant"
+	destination
+		government "Remnant"
+	on offer
+		require "Jump Drive (Broken)"
+		conversation
+			`As the <ship> settles onto its designated landing pad you see someone emerge from the nearby outfitter and quickly make their way over to your ship. They wait a safe distance back, then approaches once you cycle the hatch and disembark. "Greetings, Captain <last>," trills the Remnant. "We saw the news that the newcomer among us was tracking down the erratic Korath ship. When you came in to land we saw on the scans that you managed to salvage its malfunctioning jump drive. Could we take it to study?"`
+			choice
+				`	(Maybe later.)`
+					goto refuse
+				`	(Yes.)`
+			`	"Excellent!" he glances at his datapad, "There are several teams working on aspects of the jump drive, and the one with the highest priority for a malfunctioning jump drive is on <planet>. I will notify them to expect you."`
```

Comment:
`"Excellent!" He`

Since you've assigned a pronoun to the Remnant here, you can do so above too.

"a malfunctioning jump drive" -> "this one"

Reply:
Changed both.